src/features/collectables/HotspotListItem.tsx

```diff
@@ -51,7 +92,7 @@ const HotspotListItem = ({ hotspot, onPress }: HotspotListItemProps) => {
           height={COLLECTABLE_HEIGHT}
           width="100%"
           source={{
-            uri: json?.image,
+            uri: metadata?.image,
             cache: 'force-cache',
```

Comment:
This only works on iOS I think and it will always use cache if there is one, never updating the cache. Just want to make sure that is what we want?

Reply:
For now all hotspot images are the same so I think its safe to leave for now